pkg/vehicle/command.go

```diff
@@ -0,0 +1,479 @@
+package vehicle
```

Comment:
What's the rationale for moving this out of the proxy package? This code is specific to (1) using an HTTP Connecor, as opposed to BLE and (2) creating an HTTP proxy client, as opposed to a native Golang client.

Reply:
Good point, my original implementation was different and was using vehicle as a interface. I can move it back